static/js/tools/stat_var/info.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { Component } from "react";
+
+class Info extends Component {
+  render(): JSX.Element {
+    return (
+      <div id="placeholder-container">
+        <h1 className="mb-4">Statistical Variable Explorer</h1>
+        <p>
+          The statistical variable explorer provides information each
+          statistical variable, such as variable metadata, variable
+          observations, etc. Select a variable from the pane in the left to get
```

Comment:
note: be consistent about referring to statistical variables as either "variable" or "statistical variable". we have tended to use the full phrase "statistical variable", but for this we can keep to it as a noun (so should be "Select a statistical variable from the pane ...")